collections/app/controllers/ImageCollectionsController.scala

```diff
@@ -48,12 +50,13 @@ class ImageCollectionsController(authenticated: Authentication, config: Collecti
     // We do a get to be able to find the index of the current collection, then remove it.
     // Given that we're using Dynamo Lists this seemed like a decent way to do it.
     // Dynamo Lists, like other lists do respect order.
-    dynamo.listGet(id, "collections") flatMap { collections =>
+    imageCollectionsStore.get(id) flatMap { collections =>
       CollectionsManager.findIndexes(path, collections) match {
         case Nil =>
           Future.successful(respondNotFound(s"Collection $collectionString not found"))
         case indexes =>
-          dynamo.listRemoveIndexes(id, "collections", indexes)
+          val updatedCollections = CollectionsManager.filterCollectionsByIndexes(indexes, collections)
```

Comment:
I wasn't able to get scanamo to do the same dynamo operation to remove list indexes: 
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.UpdateExpressions.html#Expressions.UpdateExpressions.REMOVE
because passing in the element with an index gave a syntax error. 

Since this method is already pulling the record from the db, I think doing the removal in memory and overriding the record should give the same result. But would be good to double check this in code.